pages/stems/index.tsx

```diff
@@ -38,14 +43,30 @@ const StemsPage: NextPage<StemsPageProps> = props => {
 								and start a new project with them.
 							</Typography>
 						</Container>
+						<Box sx={styles.icons}>
+							<ButtonGroup color="info" variant="outlined" aria-label="outlined button group">
+								<Button variant={!listView ? 'contained' : 'outlined'} onClick={() => showListView(!listView)}>
+									<AppsIcon />
+								</Button>
+								<Button variant={listView ? 'contained' : 'outlined'} onClick={() => showListView(!listView)}>
```

Comment:
This could be abstracted to a single function and re-used rather than repeating.

```js
const handleToggleListView = () => setListView(!listView)

// and buttons a bit more readable

<Button variant={listView ? 'outlined' : 'contained'} onClick={handleToggleListView}>
  <AppsIcon />
</Button>
<Button variant={listView ? 'contained' : 'outlined'} onClick={handleToggleListView}>
  <FormatListBulletedIcon />
</Button>
```